ngraph/test/op_eval/color_convert_nv12.cpp

```diff
@@ -0,0 +1,173 @@
+                          std::vector<uint8_t>{0x91, 0x91, 0x51, 0x51, 0x91, 0x91, 0x51, 0x51, 0x29, 0x29, 0x91, 0x5f,
+                                               0x29, 0x29, 0x29, 0x29, 0x22, 0x36, 0xf0, 0x5a, 0x6e, 0xf0, 0x76, 0x9f},
+                          std::vector<uint8_t>{0x00, 0x00, 0xff, 0xff, 0x00, 0x00, 0xff, 0xff, 0x00, 0x00, 74,   132,
+                                               0x00, 0x00, 0x1b, 0x1b, 0xff, 0xff, 0x00, 0x00, 0xff, 0xff, 0x00, 0x00,
+                                               0x00, 0x00, 87,   145,  0x00, 0x00, 0x18, 0x18, 0x00, 0x00, 0x00, 0x00,
+                                               0x00, 0x00, 0x00, 0x00, 0xff, 0xff, 153,  211,  0xff, 0xff, 0x5a, 0x5a}},
+    NV12ConvertTestParams{"red_2x2_bgr",
+                          op::v8::ConvertColorNV12::ColorFormat::NV12_TO_BGR,
+                          Shape{1, 1, 3, 2},
+                          Shape{1, 3, 2, 2},
+                          std::vector<uint8_t>{0x51, 0x51, 0x51, 0x51, 0xf0, 0x5a},
+                          std::vector<uint8_t>{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xff, 0xff, 0xff, 0xff}},
+    NV12ConvertTestParams{"green_2x2_bgr",
+                          op::v8::ConvertColorNV12::ColorFormat::NV12_TO_BGR,
+                          Shape{1, 1, 3, 2},
+                          Shape{1, 3, 2, 2},
+                          std::vector<uint8_t>{0x91, 0x91, 0x91, 0x91, 0x22, 0x36},
+                          std::vector<uint8_t>{0x0, 0x0, 0x0, 0x0, 0xff, 0xff, 0xff, 0xff, 0x0, 0x0, 0x0, 0x0}},
+    NV12ConvertTestParams{"blue_2x2_bgr",
+                          op::v8::ConvertColorNV12::ColorFormat::NV12_TO_BGR,
+                          Shape{1, 1, 3, 2},
+                          Shape{1, 3, 2, 2},
+                          std::vector<uint8_t>{0x29, 0x29, 0x29, 0x29, 0x6e, 0xf0},
+                          std::vector<uint8_t>{0xff, 0xff, 0xff, 0xff, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}},
+    NV12ConvertTestParams{
+        "color_4x4_bgr",
+        op::v8::ConvertColorNV12::ColorFormat::NV12_TO_BGR,
+        Shape{1, 1, 6, 4},
+        Shape{1, 3, 4, 4},
+        std::vector<uint8_t>{0x91, 0x91, 0x51, 0x51, 0x91, 0x91, 0x51, 0x51, 0x29, 0x29, 0x91, 0x5f,
+                             0x29, 0x29, 0x29, 0x29, 0x22, 0x36, 0xf0, 0x5a, 0x6e, 0xf0, 0x76, 0x9f},
+        std::vector<uint8_t>{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xff, 0xff, 153,  211,
+                             0xff, 0xff, 0x5a, 0x5a, 0xff, 0xff, 0x00, 0x00, 0xff, 0xff, 0x00, 0x00,
+                             0x00, 0x00, 87,   145,  0x00, 0x00, 0x18, 0x18, 0x00, 0x00, 0xff, 0xff,
+                             0x00, 0x00, 0xff, 0xff, 0x00, 0x00, 74,   132,  0x00, 0x00, 0x1b, 0x1b}}};
+
+struct PrintToDummyParamName {
+    template <class ParamType>
+    std::string operator()(const ::testing::TestParamInfo<ParamType>& info) const {
+        return "color_convert_" + info.param.m_name;
+    }
+};
```

Comment:
@ilyachur should not this test be a part of template plugin func tests? I believe tests for reference impl for operations are now there

Reply:
Yes we shouldn't add new eval or backend tests

Reply:
Done